packages/orchestration/test/examples/stake-ica.contract.test.ts

```diff
@@ -102,7 +102,10 @@ test('makeAccount, getAddress', async t => {
     t.like(chainAddress, { chainId: 'osmosis-1' });
   }
 
-  t.snapshot(bootstrap.storage.data.entries(), 'accounts in vstorage');
+  t.snapshot(
+    Object.fromEntries(bootstrap.storage.data.entries()),
```

Comment:
I'm curious, what motivated this change?

Reply:
Written in the commit: https://github.com/Agoric/agoric-sdk/pull/11358/commits/0e64cd3deeabcc61d8d38f9761dec1cfd33f99f1